gds_py/Dockerfile

```diff
@@ -8,8 +8,11 @@ ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
 
 #--- Python ---#
 
-RUN conda update --yes conda 
-RUN conda install -c conda-forge -c defaults --quiet --yes \
+RUN conda update conda --yes \
```

Comment:
The less docker commands you have the less layers and smaller images you'll get. Try to "chain" the commands as much as possible.